tools/dependencies/openblas.sh

```diff
@@ -33,12 +34,24 @@ if [[ ! -e $DEPS_PATH/lib/libopenblas.a ]]; then
 
     # Adding NO_DYNAMIC=1 flag causes make install to fail
     $MAKE DYNAMIC_ARCH=1 USE_OPENMP=1
+
+    if [[ -v CMAKE_STATICBUILD ]]; then
+        # We link and redistribute libopenblas.so for cmake staticbuild
+        # cf https://gitlab.kitware.com/cmake/cmake/issues/16221#note_143330
+        patchelf --set-rpath '$ORIGIN' --force-rpath libopenblas.so
```

Comment:
why not link with the static library?

Reply:
The outcome is equivalent and the current solution is easier to support in cmake, as `libopenblas.so` declares it's dependencies correctly unlike `.a` files.

We could reconsider after fixing https://gitlab.kitware.com/cmake/cmake/issues/16221